eng/Publishing.props

```diff
@@ -47,8 +47,13 @@
     <ManifestBuildData Include="AzureDevOpsBranch=$(BUILD_SOURCEBRANCH)" />
   </ItemGroup>
 
+  <!-- The PGO sdk should always have External visibility, even if someone changes the default artifact visibility -->
   <ItemGroup>
-    <ToolsetAssetsToPublish Include="$(ArtifactsNonShippingPackagesDir)dotnet-toolset*.zip" />
+    <Artifact Update="$(ArtifactsNonShippingPackagesDir)/dotnet-sdk-pgo-*" Visibility="External" />
```

Comment:
Are the PGO archives externally shipping? I thought that they are only needed for dotnet-optimization?

Reply:
They're External visibility and NonShipping. So they get published to BAR but not stable versioned.